### PR TITLE
new implementations

### PR DESCRIPTION
**Pull request by felipeCordeiroDosSantos**

### CODE REVIEW

File: modules/connectDB.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const knex = require('knex')({
```
Comment (wesdeveloper) [inline modules/connectDB.js]: I recommend you to use environment variables... Not set fix on the code

File: modules/modulesDB.js [hunk 2]
```diff
@@ -0,0 +1,142 @@
+const knex = require('./connectDB')
+
+function addEmployee(name, email, password){
+    const newEmployee = {
+        name: name,
```
Comment (wesdeveloper) [inline modules/modulesDB.js]: You can use shorthand variables when the name is the same...

**Comment by wesdeveloper** [comment]
I recommend you to use airbnb eslint to enforce yourself to follow some code rules that are considered the best practices.

File: modules/modulesDB.js [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+    }
+
+    knex.insert(newEmployee).into('employees')
+    .then(function (data){
+        console.log(data)
+    })
+    .catch(function (error){
+        console.log(error)
+    })
+    .then(function (){
```
Comment (SeuZeRicardo) [inline modules/modulesDB.js]: Why this is empty?
It's really necessary?

File: modules/modulesDB.js [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+    }
+
+    knex.insert(newCustomer).into('customers')
+    .then(function (data){
+        console.log(data)
+    })
+    .catch(function (error){
+        console.log(error)
+    })
+    .then(function (){
```
Comment (SeuZeRicardo) [inline modules/modulesDB.js]: Why this is empty?
It's really necessary?

File: modules/modulesDB.js [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+    }
+
+    knex.insert(newLoan).into('loans')
+    .then(function (data){
+        console.log(data)
+    })
+    .catch(function (error){
+        console.log(error)
+    })
+    .then(function (){
```
Comment (SeuZeRicardo) [inline modules/modulesDB.js]: Why this is empty?
It's really necessary?

File: app.js [hunk 6]
```diff
@@ -9,7 +9,20 @@ const usersRouter = require('./routes/users');
 
 const app = express();
 
-const db = require('./modules/modulesDB')
+// eslint-disable-next-line import/order
+const passport = require('passport');
```
Comment (wesdeveloper) [inline app.js:12]: You need to fix the eslint, this rule really makes sense.

File: auth.js [hunk 7]
```diff
@@ -0,0 +1,45 @@
+const bcrypt = require('bcryptjs');
+const LocalStrategy = require('passport-local').Strategy;
+
+const knex = require('./connectDB');
```
Comment (wesdeveloper) [inline auth.js]: If you are making an import of connectDB a good variable name would be **databaseConnection**

File: modules/books.js [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+function listBooks() {
+  // eslint-disable-next-line no-unused-expressions
+  knex.select(['name', 'author', 'year', 'area', 'edition', 'publisher']).table('books')
+    .then((data) => {
+      console.log(data);
+    })
+    .catch((error) => {
+      console.log(error);
+    })
+    .then;
```
Comment (wesdeveloper) [inline modules/books.js:33]: What???

File: modules/books.js [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+}
+
+function listBooks() {
+  // eslint-disable-next-line no-unused-expressions
+  knex.select(['name', 'author', 'year', 'area', 'edition', 'publisher']).table('books')
```
Comment (wesdeveloper) [inline modules/books.js]: Do you will use the data only inside the functions listBooks?

File: routes/index.js [hunk 10]
```diff
@@ -1,9 +1,47 @@
...
 
-/* GET home page. */
-router.get('/', function(req, res, next) {
-  res.render('index', { title: 'Express' });
+const passport = require('passport');
+
+const books = require('../controller/booksController');
+
+router.get('/', (req, res, next) => {
+  res.render('login');
```
Comment (wesdeveloper) [inline routes/index.js]: I really recommend you to always use "return" on res.send or res.render to finish the code execution....

`return   res.render('login');`

**Pull request merged**